src/BenchmarksServer/Startup.cs

```diff
@@ -289,7 +273,7 @@ private static void DeleteDir(string path)
         private static Process StartProcess(string hostname, string benchmarksRepo, ServerJob job)
         {
             var filename = "dotnet";
-            var arguments = $"run -c Release -- --scenarios {job.Scenario} --server {job.WebHost} " +
+            var arguments = $"run -c Release -f netcoreapp1.1 -- --scenarios {job.Scenario} --server {job.WebHost} " +
```

Comment:
Yes, if the project targets multiple TFMs. It doesn't pick one up by default anymore.